examples/ivr/dialer.py

```diff
@@ -80,16 +80,20 @@ def create_url():
         # Adding F at the end of the DID disables remote SS7 overlap dialing
         # which can add 5 sec to the incoming call setup time.
         return {
-            'dest_url': 'a/4113',
+            'dest_url': 'a/1000F',
             'dest_profile': 'g1',
-            'dest_endpoint': 'freetdm'
+            'dest_endpoint': 'freetdm',
+            'caller_id': '\'Sangoma Technologies\'',
```

Comment:
Reason I stuck with ' instead of "
As you can see all above strings are using ' 

Nenad